pkgs/development/python-modules/protobuf/default.nix

```
@@ -1,5 +1,6 @@
 { buildPackages
 , lib
+, buildPackagesProtobuf ? buildPackages.protobuf
```

```suggestion
, buildPackagesProtobuf
```
Please move this to top-level.

---

this was removed, instead using version as a way to find the correct package